stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/attachments/ChannelMediaAttachmentsGrid.kt

```diff
@@ -156,14 +156,14 @@ internal fun ChannelMediaAttachmentsGrid(
         ) {
             itemsIndexed(
                 items = content.items,
-                key = { _, item -> item.id },
+                key = { index, item -> "${item.id}-$index " },
             ) { index, item ->
                 itemContent(index, item) {
                     previewItemIndex = index
                 }
             }
             if (content.isLoadingMore) {
-                item { loadingItem() }
+                item(key = "loading_item") { loadingItem() }
```

Comment:
I believe the root cause is not in the `ChannelMediaAttachmentsGrid` file. It is in the `item.id` calculation.

The error report mentioned: `"-1891661558-Loading" was already used.`
Which means, according to the current logic, `Loading` is the attachment displayable name, which can be `title ?: name ?: upload?.name`

A more stable ID would be using `imagePreviewUrl` instead

```kotlin
public val id: String = "${message.identifierHash()}-${attachment.imagePreviewUrl}"
```

PS.: The key for loading item (`item(key = "loading_item")`) seems to be irrelevant for the issue.